app/assets/javascripts/modules/FilterTable.js

```diff
@@ -65,6 +65,18 @@ define(['jquery', 'DoughBaseComponent', 'List', 'ListFuzzySearch'],
     return $.unique(filterClasses);
   };
 
+  /**
+   * enableFuzzySearch
+   *
+   * Find input field with class filterFieldClass then add fuzzy-search class to it.
+   *
+   */
+  FilterTableProto.enableFuzzySearch = function () {
+    this.$el.find('.' + this.config.filterFieldClass).each(function(_, cell) {
```

Comment:
Good catch actually — they're not cells either. Since this just got merged I'll open a quick PR for it.